com/course/zenkouv/src/Lecture3/Task1/KnightApp.java

```diff
@@ -0,0 +1,33 @@
+package Lecture3.Task1;
+
+import Lecture3.Task1.model.*;
+
+class KnightApp {
+
+    private static final Knight artur = new Knight("Artur");
+    private static final int MAX_COST = 12;
+    private static final int MIN_COST = 10;
+
+    public static void main(String[] args) {
+        artur.addOutfitToOutfitLis(new Weapon(16, 5.5, Material.STEEL, 25, WeaponType.SWORD));
```

Comment:
move creation of such elements like `Knight ` and `Weapon ` to KnightFactory and OutfitFactory with static factory methods

Reply:
fix

Reply:
fix